sites/example-project/src/pages/api/pagesManifest.json/+server.js

```diff
@@ -63,12 +64,14 @@ export async function GET() {
 	try {
 		const pages = {};
 
-		const pagesDir = fs.readdirSync('src/pages', { withFileTypes: true, recursive: true });
+		// for some reason `readDirSync` returns different results (on windows)??
+		const pagesDir = await fs.readdir('src/pages', { withFileTypes: true, recursive: true });
 		for (const dirent of pagesDir) {
 			if (dirent.isFile() && dirent.name.endsWith('.md')) {
-				const path = `${dirent.parentPath ?? dirent.path}/${dirent.name}`;
-				const content = fs.readFileSync(path, 'utf-8');
-				pages[`/${path}`] = content;
+				const relative_path = path.join(dirent.parentPath ?? dirent.path, dirent.name);
+				const content = await fs.readFile(relative_path, 'utf-8');
+				// regularize for windows
+				pages[new URL(`file:///${relative_path}`).pathname] = content;
```

Comment:
I also just ran a test for this change, and both generated trees were identical. What exactly does this fix?

Reply:
Were you running it on windows? should be different on windows vs linux w/o the `new URL` thing

Reply:
I was indeed running on linux, not Windows 🙃. I'll check again.

Reply:
Confirmed this is a good fix

![image](https://github.com/user-attachments/assets/3de628a1-fdad-4b15-81a5-8970ec3ffe49)

Reply:
(left is before, right is after)